samples/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would add this as a module to the parent pom so it gets built and updated in a normal build

Reply:
We want the samples to be independent.